engine/chainstate/mock_const.go

```diff
@@ -29,5 +29,5 @@ const (
 // MockDefaultFee is a mock default fee used for assertions
 var MockDefaultFee = &utils.FeeUnit{
 	Satoshis: 1,
-	Bytes:    20,
+	Bytes:    1000,
```

Comment:
Just wondering why need to change this value ?

Reply:
I guess, It's the value which TALL-ARC currently offers.

Reply:
yes, this is what taal offers, the same values are in config.examle.yaml